src/libraries/Native/Unix/System.Security.Cryptography.Native.Apple/pal_digest.c

```diff
@@ -226,3 +208,31 @@ int32_t AppleCryptoNative_DigestOneShot(PAL_HashAlgorithm algorithm, uint8_t* pB
             return -1;
     }
 }
+
+int32_t AppleCryptoNative_DigestReset(const DigestCtx* ctx)
```

Comment:
"Reset" is definitely not "const" :)

Reply:
1AM coding isn't as effective as it used to be. 💤